server/docker/config_template.yaml

```diff
@@ -45,5 +45,6 @@ auth:
 codec:
   endpoint: {{ default .Env.TEMPORAL_CODEC_ENDPOINT "" }}
   passAccessToken: {{ default .Env.TEMPORAL_CODEC_PASS_ACCESS_TOKEN "false" }}
+  includeCredentials: {{ default .Env.TEMPORAL_CODEC_INCLUDE_CREDENTIALS "false" }}
```

Comment:
I don't quite agree with supporting this everywhere maybe we can add something like "UNSUPPORTED" Or "DANGEROUS"?

Reply:
I'll let @robholland and @feedmeapples weigh in on the naming for this env var